tests/test_integrated_channels/test_sap_success_factors/test_client.py

```diff
@@ -242,21 +259,19 @@ def test_sap_api_application_error(self):
 
     @responses.activate
     def test_expired_access_token(self):
+        """
+           If our token expires after some call, make sure to get it again.
+
+           Make a call, have the token expire after waiting some time (technically no time since time is frozen),
+           and make a call again and notice 2 OAuth calls in total are required.
+        """
         expired_token_response_body = {"expires_in": 0, "access_token": self.access_token}
         responses.add(
             responses.POST,
             self.url_base + self.oauth_api_path,
             json=expired_token_response_body,
             status=200
         )
-
```

Comment:
why we need to remove this `responses` block.

Reply:
It was not mandatory to remove this block.
I removed this block as it was not necessary to add a new response as it doesn't matter if 2nd oauth values are going to expire instantly or not.